pom.xml

```diff
@@ -66,6 +74,13 @@
             <artifactId>problem-spring-web</artifactId>
             <version>${zalando.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>postgresql</artifactId>
+            <version>1.15.2</version>
```

Comment:
[minor] Consider to move version to the properties the same as for junit test container

Reply:
sure, thanks, missed this one.